var/spack/repos/builtin/packages/hdf5-vol-async/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hdf5VolAsync(CMakePackage):
+    """This package enables asynchronous IO in HDF5."""
+
+    homepage = "https://sdm.lbl.gov/"
+    git      = "https://github.com/hpc-io/vol-async"
+    maintainers = ['hyoklee']
+
+    version('develop', branch='develop')
```

Comment:
Add a different version until https://github.com/hpc-io/vol-async/pull/13 is accepted.
```
    version('develop', branch='develop')
    version('hyoklee.develop',
            branch='develop',
            git='https://github.com/hyoklee/vol-async', preferred=True)
```